101-app-configuration-store-keyvaultref/azuredeploy.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "configStoreName": {
+      "type": "string",
+      "metadata": {
+        "description": "Specifies the name of the App Configuration store."
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Specifies the Azure location where the app configuration store should be created."
+      }
+    },
+    "keyValueName": {
+      "type": "string",
+      "defaultValue": "KeyVaultReferenceSample",
+      "metadata": {
+        "description": "Specifies the name of the key-value resource. The name is a combination of key and label with $ as delimiter."
+      }
+    },
+    "keyValueSecretIdentifier": {
+      "type": "string",
+      "defaultValue": "https://your-key-valut-name.vault.azure.net/secrets/your-secret",
```

Comment:
remove this since it won't work for anyone... you can supply a value in the param file as needed.

Reply:
User can store whatever string they prefer and do the string interpretation on client side. App Config service doesn't check the string on server side. But you make a point. I use the `GEN-KEYVAULT-ENCRYPTION-KEY-URI` placeholder to create a meaningful URI to make sure CI can pass.